--- a/src/dust.rs
+++ b/src/dust.rs
@@ -1326,10 +1326,10 @@
 #[storable(db = D)]
 #[tag = "dust-local-state[v1]"]
 pub struct DustLocalState<D: DB> {
-    generating_tree: MerkleTree<DustGenerationInfo, D>,
-    generating_tree_first_free: u64,
-    commitment_tree: MerkleTree<(), D>,
-    commitment_tree_first_free: u64,
+    pub generating_tree: MerkleTree<DustGenerationInfo, D>,
+    pub generating_tree_first_free: u64,
+    pub commitment_tree: MerkleTree<(), D>,
+    pub commitment_tree_first_free: u64,
     night_indices: HashMap<InitialNonce, u64, D>,
     dust_utxos: HashMap<DustNullifier, DustWalletUtxoState, D>,
     pub sync_time: Timestamp,
